core/jreleaser-model-impl/src/main/java/org/jreleaser/model/internal/release/Changelog.java

```diff
@@ -940,7 +940,11 @@ public boolean containsContributor(String name) {
             if (isNotBlank(name)) {
                 String n = name.trim();
                 for (String contributor : contributors) {
-                    if (n.contains(contributor) || n.matches(toSafeRegexPattern(contributor))) {
+                    if(n.endsWith("bot")) {
```

Comment:
These 3 lines are the ones I'm not so sure about as this behavior is not configurable at the moment and in some cases it might filter a valid account with no way to "unfilter". I think it's better if users to explicitly skip bots for the time being. Exposing a configuration option for this might be a bit messy with what we already have in place.

Thus I think the minimum fix is to remove these 3 lines and keep the change you made in line 947. Thank you 😅

Reply:
I wasn't quite sure if I am missing a simple solution to receiving the context in this class, thus I initially implemented it this way. Your answer confirms my 'fear' of not being able to receive context/ environment information right now. 

I removed the `endsWith` check. Hopefully you find an easy way to integrate the bot-check in an easy manner :)